### PR TITLE
manifest: Bring nrfxlib with CC3XX PSA fixes

### PR DESCRIPTION
**Pull request by Vge0rge**
Brings nrfxlib with a correction on the supported algorithms when CC3XX is used with PSA APIs. AES OFB is now reported as supported and the CFB is reported as not supported as it should.

Ref: NCSDK-21022
Ref: NCSDK-20664

### CODE REVIEW

**Review by SebastianBoe — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Comment by joerchan** [comment]
I got confused about what you're fixing, but you're fixing docs :P

Maybe change commit title to "doc fixes" when you rebase for the real commit SHA?

File: west.yml [hunk 1]
```diff
@@ -131,7 +131,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 508e546ee92bf37100425bfef48418f66fe4215e
+      revision: pull/977/head
```
Comment (rlubos) [inline west.yml:134]: Please update SHA: `37e3dd8997b3db49527c7cdcea92021ee3c0e97c` (and rebase).

**Comment by Vge0rge** [comment]
> I got confused about what you're fixing, but you're fixing docs :P
> 
> Maybe change commit title to "doc fixes" when you rebase for the real commit SHA?

The doc fix is a minor thing of this PR.  The commit that brings nrfxlib fixes an issue that we had where AES CFB was reported as supported even though we don't support the algorithm and AES OFB was reported as not supported even though we support it. That's a fix and not just a doc update in my opinion. 

The minor doc fix in the tfm_psa_template sample is so small that I didn't think of opening a new PR for this. That's why I included it here.

**Review by joerchan — APPROVED** [review]

**Comment by rlubos** [comment]
CI finally passed after Matter instability has been fixed.

**Pull request merged**